# <span style="font-size: 20pt; font-family: Montserrat, sans-serif; color: rgb(92, 148, 206); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Graded Assignment 3: 9 to 5</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Time to show off your SQL skills! For each question, copy the SQL query you used and make note of the answer.</span>

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">The Dataset</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">For this assignment, you will be using the Bureau of Labor Statistics (BLS) Current Employment Survey (CES) results which can be found on</span> [<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(17, 85, 204); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; text-decoration-line: underline; vertical-align: baseline; white-space: pre-wrap;">Kaggle</span>](https://www.kaggle.com/bls/employment)<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.</span>

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Business Issue</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">You are working for the Bureau of Labor Statistics with the United States government and have been approached by your boss with an important meeting request. You have been asked by your supervisor to meet with Dolly Parton whose nonprofit is looking to shed light on the state of employment in the United States. As part of the 9 to 5 project, their research is focused on production and nonsupervisory employees and how those employees fare compared to all employees in the United States. While the data the BLS collects from the CES is publicly available, Dolly Parton and her colleagues need your assistance navigating the thousands of rows in each table in LaborStatisticsDB.</span>

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">About the Dataset</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">This dataset comes directly from the Bureau of Labor Statistics’ Current Employment Survey (CES). Here are some things you need to know:</span>

1. <span style="font-size: 10pt; background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">The industry table contains an NAICS code. This is different from the industry code. NAICS stands for North American Industry Classification System.</span>
2. <span style="font-size: 10pt; background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Series ID is composed of multiple different codes. CES stands for Current Employment Survey, the name of the survey which collected the data. The industry code as specified by the BLS and the data type code as specified in the datatype table.</span>

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Set Up</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">To connect to the database, use the same connection info used during the SQL lessons.&nbsp;</span> 

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">For the assignment, we will be using the LaborStatisticsDB database.</span>

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Database Exploration</span>

<span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">To start with, let’s get to know the database further.</span>

1. <span style="background-color: rgb(255, 255, 255); color: rgb(102, 102, 102); font-family: &quot;Times New Roman&quot;; font-size: 10pt; white-space: pre-wrap;">Use this space to make note of each table in the database, the columns within each table, each column’s data type, and how the tables are connected. You can write this down or draw a diagram. Whatever method helps you get an understanding of what is going on with LaborStatisticsDB.</span>

<span style="font-size: 13.3333px; white-space: pre-wrap;"><br></span>

![diagram](schema_diagram.png)

2. <span style="background-color: rgb(255, 255, 255); color: rgb(102, 102, 102); font-family: Arvo, serif; font-size: 10pt; white-space: pre-wrap;">What is the datatype for women employees?</span>

In [1]:
SELECT data_type_code 
FROM LaborStatisticsDB.dbo.datatype 
WHERE data_type_text = 'women employees';

USE LaborStatisticsDB;
SELECT DATA_TYPE 
FROM INFORMATION_SCHEMA.COLUMNS 
WHERE TABLE_NAME = 'datatype'
AND COLUMN_NAME = 'data_type_text';

-- Data_Type_Code is 10 for Women Employees
-- Data type for that column is NVARCHAR

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.181

data_type_code 10

DATA_TYPE nvarchar

3. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">What is the series id for&nbsp; women employees in the commercial banking industry in the financial activities supersector?</span>

In [2]:
SELECT *
FROM LaborStatisticsDB.dbo.series 
WHERE data_type_code = (SELECT data_type_code FROM datatype WHERE data_type_text = 'women employees')
AND industry_code = (SELECT industry_code FROM industry WHERE industry_name = 'commercial banking')
AND supersector_code = (SELECT supersector_code FROM supersector WHERE supersector_name = 'financial activities')

-- CES5552211010 for Seasonally Adjusted 
-- CEU5552211010 for Not Seasonally Adjusted

(2 rows affected)

Total execution time: 00:00:00.152

series_id supersector_code industry_code data_type_code seasonal series_title CES5552211010 55 55522110 10 S Women employees CEU5552211010 55 55522110 10 U Women employees

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Join in on the Fun</span>

<span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Time to start joining! You can choose the type of join you use, just make sure to make a&nbsp; note!</span>

1. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join&nbsp; annual_2016 with series on series_id. We only want the data in the annual_2016 table to be included in the result.</span>

In [ ]:
-- ***
-- If my queries return a lot of rows, I will use 'SELECT TOP(5)' to make it easy to scroll through this document.
-- ***

In [3]:
SELECT TOP 5 * 
FROM LaborStatisticsDB.dbo.annual_2016 
LEFT JOIN LaborStatisticsDB.dbo.series 
ON annual_2016.series_id = series.series_id

-- In this case, LEFT join will keep all the data from annual_2016

(5 rows affected)

Total execution time: 00:00:00.483

id series_id year period value footnote_codes original_file series_id supersector_code industry_code data_type_code seasonal series_title 0 CEU5500000007 2016 M13 36.9 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000007 55 55000000 7 U Average weekly hours of production and nonsupervisory employees 1 CEU5500000008 2016 M13 26.11 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000008 55 55000000 8 U Average hourly earnings of production and nonsupervisory employees 2 CEU5500000030 2016 M13 962.73 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000030 55 55000000 30 U Average weekly earnings of production and nonsupervisory employees 3 CEU5500000031 2016 M13 411.29 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000031 55 55000000 31 U Average weekly earnings of production and nonsupervisory employees 4 CEU5500000032 2016 M13 11.15 NULL ce.data.55c.FinancialActivities.ProductionEmployeeHoursAndEarnings.csv CEU5500000032 55 55000000 32 U Average hourly earnings of production and nonsupervisory employees

2. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join series and datatype on data_type_code</span>

In [4]:
SELECT TOP 5 * 
FROM LaborStatisticsDB.dbo.series 
JOIN LaborStatisticsDB.dbo.datatype 
ON series.data_type_code = datatype.data_type_code

-- INNER join will combine rows that have matching values in both tables

(5 rows affected)

Total execution time: 00:00:00.097

series_id supersector_code industry_code data_type_code seasonal series_title data_type_code data_type_text CES0000000001 0 00000000 1 S All employees 1 ALL EMPLOYEES CES0000000010 0 00000000 10 S Women employees 10 WOMEN EMPLOYEES CES0000000025 0 00000000 25 S All employees 25 ALL EMPLOYEES CES0000000026 0 00000000 26 S All employees 26 ALL EMPLOYEES CES0500000001 5 05000000 1 S All employees 1 ALL EMPLOYEES

3. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Join series and industry on industry_code</span>

In [5]:
SELECT TOP 5 * 
FROM LaborStatisticsDB.dbo.series 
JOIN LaborStatisticsDB.dbo.industry 
ON series.industry_code = industry.industry_code

-- INNER join will combine rows that have matching values in both tables

(5 rows affected)

Total execution time: 00:00:00.110

series_id supersector_code industry_code data_type_code seasonal series_title id industry_code naics_code publishing_status industry_name display_level selectable sort_sequence CES0000000001 0 00000000 1 S All employees 0 0 - B Total nonfarm 0 T 1 CES0000000010 0 00000000 10 S Women employees 0 0 - B Total nonfarm 0 T 1 CES0000000025 0 00000000 25 S All employees 0 0 - B Total nonfarm 0 T 1 CES0000000026 0 00000000 26 S All employees 0 0 - B Total nonfarm 0 T 1 CES0500000001 5 05000000 1 S All employees 1 5000000 - A Total private 1 T 2

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Aggregate Your Friends and Code some SQL</span>

<span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Put together the following:</span>

1. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many employees were reported in 2016 in all industries? Round to the nearest whole number.&nbsp; No chopping people into little bits, please.</span>

In [6]:
-- Before I started, I noticed one of the original files ('ce.data.0.AllCESSeries.csv') creates duplicate information.
-- Below I compared the total employee count by only including the file and excluding the file from the count. 
-- The total for both is the same, which confirms that it is duplicate data.

USE LaborStatisticsDB;
SELECT ROUND(SUM(value),0) AS 'Total Employee Count (only inculding file)'
FROM annual_2016
WHERE series_id IN (SELECT series_id FROM series WHERE series_title = 'all employees')
AND original_file = 'ce.data.0.AllCESSeries.csv';

SELECT ROUND(SUM(value),0) AS 'Total Employee Count (excluding file)'
FROM annual_2016
WHERE series_id IN (SELECT series_id FROM series WHERE series_title = 'all employees')
AND original_file != 'ce.data.0.AllCESSeries.csv';

-- So there were 1,170,306 employees reported in 2016 for all industries.

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.286

Total Employee Count (only inculding file) 1170306

Total Employee Count (excluding file) 1170306

2. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many women employees were reported in 2016 in all industries? Round to the nearest whole number.&nbsp;</span>

In [7]:
SELECT ROUND(SUM([value]),0) AS 'Total Women Employee Count'
FROM LaborStatisticsDB.dbo.annual_2016
WHERE series_id IN (SELECT series_id FROM LaborStatisticsDB.dbo.series WHERE series_title = 'women employees')
AND original_file = 'ce.data.0.AllCESSeries.csv';

-- 562,745 women employees were reported in 2016 in all industries
-- Indicated the file, or else the duplicated will affect the total sum

(1 row affected)

Total execution time: 00:00:00.131

Total Women Employee Count 562745

3. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">How many production/nonsupervisory employees were reported in 2016? Round to the nearest whole number.&nbsp;</span>

In [8]:
SELECT ROUND(SUM([value]),0) AS 'Total Production/Nonsupervisory Employee Count'
FROM LaborStatisticsDB.dbo.annual_2016
WHERE series_id IN (SELECT series_id FROM LaborStatisticsDB.dbo.series WHERE series_title = 'production and nonsupervisory employees')
AND original_file = 'ce.data.0.AllCESSeries.csv';

-- 631,825 production/nonsupervisory employees were reported in 2016

(1 row affected)

Total execution time: 00:00:00.161

Total Production/Nonsupervisory Employee Count 631825

4. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2017, what is the average weekly hours worked by production and nonsupervisory employees across all industries?</span>

In [9]:
SELECT ROUND(AVG([value]),2) AS 'Average Weekly Hours - Production/Nonsupervisory Employees'
FROM LaborStatisticsDB.dbo.january_2017
WHERE series_id IN (
    SELECT series_id FROM series 
    WHERE series_title = 'Average weekly hours of production and nonsupervisory employees');

-- 36 hours was the approximate average weekly hours worked by production and nonsupervisory employees in January 2017.
-- Didn't need to specify the duplicate in this case because the average will be the same

(1 row affected)

Total execution time: 00:00:00.147

Average Weekly Hours - Production/Nonsupervisory Employees 36.06

5. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">What is the total weekly payroll for production and nonsupervisory employees across all industries in January 2017? Round to the nearest penny, please.</span>

In [10]:
SELECT ROUND(SUM([value]),2) AS 'Total Weekly Payroll - Production/Nonsupervisory Employees'
FROM LaborStatisticsDB.dbo.january_2017
WHERE series_id IN (
    SELECT series_id FROM series 
    WHERE series_title = 'Aggregate weekly payrolls of production and nonsupervisory employees')
AND original_file = 'ce.data.0.AllCESSeries.csv';

-- 919376610 total weekly payroll for the production and nonsupervisory employees.

(1 row affected)

Total execution time: 00:00:00.157

Total Weekly Payroll - Production/Nonsupervisory Employees 919376610

6. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2017, which industry was the average weekly hours worked by production and nonsupervisory employees the highest? Which industry was the lowest?</span>

In [11]:
USE LaborStatisticsDB;
SELECT TOP(1) industry_name, (ROUND(AVG([value]),2)) AS avg_weekly_hrs
FROM january_2017
JOIN series ON series.series_id=january_2017.series_id
JOIN industry ON industry.industry_code=series.industry_code
WHERE series_title = 'Average weekly hours of production and nonsupervisory employees'
GROUP BY industry_name
ORDER BY avg_weekly_hrs DESC;

SELECT TOP(1)industry_name, (ROUND(AVG([value]),2)) AS avg_weekly_hrs
FROM january_2017
JOIN series ON series.series_id=january_2017.series_id
JOIN industry ON industry.industry_code=series.industry_code
WHERE series_title = 'Average weekly hours of production and nonsupervisory employees'
GROUP BY industry_name
ORDER BY avg_weekly_hrs ASC;

-- Highest: Motor Vehicle Power Train Components
-- Lowest: Fitness and Recreational Sports Centers

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.219

industry_name avg_weekly_hrs Motor vehicle power train components 49.6

industry_name avg_weekly_hrs Fitness and recreational sports centers 16.85

7. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">In January 2017, which industry was the total weekly payroll for production and nonsupervisory employees the highest? Which industry was the lowest?</span>

In [12]:
USE LaborStatisticsDB;
SELECT TOP(1) industry_name, (ROUND(SUM([value]),2)) AS total_weekly_payroll
FROM january_2017
JOIN series ON series.series_id=january_2017.series_id
JOIN industry ON industry.industry_code=series.industry_code
WHERE series_title = 'Aggregate weekly payrolls of production and nonsupervisory employees'
    AND original_file = 'ce.data.0.AllCESSeries.csv' -- because we're taking the sum, duplicates will affect our total
GROUP BY industry_name
ORDER BY total_weekly_payroll DESC; -- Descending to see the highest on top

SELECT TOP(1)industry_name, (ROUND(SUM([value]),2)) AS total_weekly_payroll
FROM january_2017
JOIN series ON series.series_id=january_2017.series_id
JOIN industry ON industry.industry_code=series.industry_code
WHERE series_title = 'Aggregate weekly payrolls of production and nonsupervisory employees'
    AND original_file = 'ce.data.0.AllCESSeries.csv' -- removing duplicate data
GROUP BY industry_name
ORDER BY total_weekly_payroll ASC; -- Ascending to see the lowest on top

-- Highest: Total Private
-- Lowest: Coin-operated Laundries and Drycleaners

(1 row affected)

(1 row affected)

Total execution time: 00:00:00.261

industry_name total_weekly_payroll Total private 147972473

industry_name total_weekly_payroll Coin-operated laundries and drycleaners 20224

## <span style="font-size: 12pt; font-family: Montserrat, sans-serif; color: rgb(53, 74, 95); background-color: rgb(255, 255, 255); font-weight: 700; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Subqueries, Unions, Derived Tables, Oh My!</span>

1. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Write a query that returns the series_id, industry_code, industry_name, and value from the january_2017 table but only if that value is greater than the average value for annual_2016 of data_type_code 82.</span>

In [13]:
USE LaborStatisticsDB;
SELECT TOP(5) jan.series_id, series.industry_code, industry.industry_name, jan.value
FROM january_2017 AS jan
JOIN series ON series.series_id = jan.series_id
JOIN industry ON industry.industry_code = series.industry_code
WHERE jan.value > (
    SELECT AVG([value]) FROM annual_2016 
    JOIN series ON series.series_id=annual_2016.series_id 
    WHERE data_type_code = 82);

-- data_type_code (82) = data_type_text (Aggregate Weekly Payrolls of Production and Nonsupervisory Employees)

(5 rows affected)

Total execution time: 00:00:00.358

series_id industry_code industry_name value CES5500000082 55000000 Financial activities 6328968 CES5552200082 55522000 Credit intermediation and related activities 1717912 CES5552210082 55522100 Depository credit intermediation 971796 CES5552300082 55523000 NULL 1044224 CES5552400082 55524000 Insurance carriers and related activities 2347328

2. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;">Create a Union table comparing average weekly earnings&nbsp; of production and nonsupervisory employees between annual_16 and january_17 using the data type 30.&nbsp; Round to the nearest penny.&nbsp; You should have a column for the average earnings and a column for the year, and the period.</span>

In [14]:
SELECT [year], [period], ROUND(AVG([value]),2) as avg_weekly_earning
FROM LaborStatisticsDB.dbo.january_2017
JOIN series ON series.series_id=january_2017.series_id
WHERE data_type_code = 30
GROUP BY [year], [period]
UNION
SELECT [year], [period], ROUND(AVG([value]),2) as avg_weekly_earning
FROM LaborStatisticsDB.dbo.annual_2016
JOIN series ON series.series_id=annual_2016.series_id
WHERE data_type_code = 30
GROUP BY [year], [period]

-- data_type_code (30) = data_type_text (Average Weekly Earnings of Production and Nonsupervisory Employees)
-- period (M13) = month (Annual Average)
-- period (M01) = month (January)

(2 rows affected)

Total execution time: 00:00:00.189

year period avg_weekly_earning 2016 M13 797.2 2017 M01 808.53

## <span style="font-size:12pt;font-family:Montserrat, sans-serif;color:rgb(53, 74, 95);background-color:rgb(255, 255, 255);font-weight:700;vertical-align:baseline;white-space:pre-wrap;">Summarize Your Results</span>

<span style="font-size:10pt;font-family:Arvo, serif;color:rgb(102, 102, 102);background-color:rgb(255, 255, 255);vertical-align:baseline;white-space:pre-wrap;">With what you know now about the </span> <span style="background-color:rgb(255, 255, 255);color:rgb(102, 102, 102);font-family:&quot;Times New Roman&quot;;font-size:13.3333px;white-space:pre-wrap;">Bureau of Labor Statistics (BLS) Current Employment Survey (CES) results and working with the</span> <span style="color:rgb(102, 102, 102);white-space:pre-wrap;font-family:&quot;Times New Roman&quot;;font-size:13.3333px;">Labor Statistics Database</span><span style="background-color:rgb(255, 255, 255);color:rgb(102, 102, 102);font-family:Arvo, serif;font-size:10pt;white-space:pre-wrap;">, answer the following questions. Note that while this is subjective, you should include relevant data to back up your opinion.</span>

1. <span style="background-color:rgb(255, 255, 255);font-size:10pt;white-space:pre-wrap;color:rgb(102, 102, 102);font-family:Arvo, serif;"><b>During which time period did production and nonsupervisory employees fare better?</b></span>

> The production and nonsupervisory employees were fared better in January 2017 with an average weekly earning of 808.53 compared to the annual average in 2016 where the average weekly earning was 797.20.

2. <span style="background-color: rgb(255, 255, 255); font-size: 10pt; white-space: pre-wrap; color: rgb(102, 102, 102); font-family: Arvo, serif;"><b>In which industries did production and nonsupervisory employees fare better?</b></span>

> As of January 2017, the total weekly payroll was highest in the 'Total Private' industry for production and nonsupervisory employees.<span style="font-size: 13.3333px; white-space: pre-wrap;"><br></span>

3. <span style="font-size: 10pt; font-family: Arvo, serif; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255); font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"><b>Now that you have explored the datasets, is there any data or information that you wish you had in this analysis?</b></span>

> It would be great to have more data on both genders and the amount each works compared to the other. To check whether one gender dominated in one industry compared to the other. I wished the Annual\_2016 table had separate data for each month - instead of just a yearly average.
> 
> In addition, I wish I knew more about the data I was working with so I could work more effeciently. I tried learning more through the Kaggle description but I think I would have worked more effeciently if I was introduced to it by someone who was an expert at the data.